### PR TITLE
Various fixes to portable-screencap

### PR DESCRIPTION
**Pull request by chriscool**
Here are some changes to portable-screencap.
It looks like many of those fixes could be aplied to ipfs-screencap too, but that will be for another PR.

### CODE REVIEW

File: portable-screencap [hunk 1]
```diff
@@ -15,30 +19,27 @@ while [ $# -gt 0 ]; do
   shift
 
   case "$arg" in
-  -h) usage ;;
-  --help) usage ;;
+  -h|--help) usage ;;
   --*)
-    die "unrecognised option: '$arg'\n$usage" ;;
+    die "unrecognised option: '$arg'\n$USAGE" ;;
```
Comment (jbenet) [inline portable-screencap:24]: whoops copy-paste fail on my part

File: portable-screencap [hunk 2]
```diff
@@ -50,15 +51,15 @@ please install a supported one:
...
 }
 
 date=$(date +"%Y-%m-%dT%H:%M:%SZ")
-fpath=$(mktemp /tmp/screencap.$date.XXXXXX.png)
-all_screencap $fpath
-cat "$fpath"
-rm "$fpath" || true
+fpath=$(mktemp "/tmp/screencap.$date.XXXXXX.png")
+all_screencap "$fpath" || die "capture failed"
+echo "Capture is in '$fpath'"
```
Comment (jbenet) [inline portable-screencap:64]: i want this script to output the file to stdout. storing in files makes other things (like chaining operations) harder. `portable-screencap >filename` is easy enough.

File: portable-screencap [hunk 3]
```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
-set -e
-set -o pipefail
```
Comment (jbenet) [inline portable-screencap:4]: i recall needing `set -o pipefail` for something. may have been in the `ipfs-screencap` script.

**Comment by jbenet** [comment]
closing in favor of #6

File: portable-screencap (hunk 2 above)
Comment (chriscool) [inline portable-screencap:64]: Ok sorry about that.

File: portable-screencap (hunk 3 above)
Comment (chriscool) [inline portable-screencap:4]: It makes something like `false | true` fail though it succeeds by default:

```
$ false | true  
$ echo $?
0
$ set -o pipefail
$ false | true
$ echo $?
1
```

But as I said in https://github.com/chriscool/ipfs-screencap/commit/fcd55a5ef00ca4ef9f095d95b1f16aac34be3161 it is not portable as dash, which is the default /bin/sh on Ubuntu, doesn't know about it:

```
$ /bin/sh
$ set -o pipefail
/bin/sh: 1: set: Illegal option -o pipefail
```

**Pull request closed without merging**